morph/react-dom.js

```diff
@@ -8,7 +8,6 @@ import toComponent from './react/to-component.js'
 import walk from './walk.js'
 
 const imports = {
-  Animated: "import Animated from 'react-dom-animated'",
```

Comment:
shouldn't we keep this one in web too?

Reply:
Tbh I wasn't sure about this, but it seemed like it's not being used anywhere?

Reply:
aren't the springs in dom using animated too?

Reply:
Correct, but not the same library. I'm using [this one](https://github.com/animatedjs/animated) instead. I'm adding the import statement for that in `get-dependencies` [here](https://github.com/viewstools/morph/blob/55ef7963fa2a75d6a82022457ab9b78f03d32a99/morph/react/get-dependencies.js#L77-L79)

Isn't `react-dom-animated` basically obsolete now?

Reply:
missed that :) that's fine then, thanks!